graph.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+# graph.py
+from collections import defaultdict
+
+
+class Graph():
+    def __init__(self, vertices):
+        self.graph = defaultdict(list)
+        self.V = vertices
+
+    def add_edge(self, u, v):
+        self.graph[u].append(v)
+    #internal
```

Comment:
If this is "internal" then I would suggest following Python style guidelines and naming the function `_is_cyclic_util` (with the preceding underscore).